test/cmdline.sh

```diff
@@ -103,11 +103,13 @@ diff -u $srcdir/test/data/en-US/expected-paraphrase2.tsv paraphrase2.tsv
 
 node $srcdir/tool/genie.js compile-ppdb -o compiled-ppdb.bin $srcdir/test/data/en-US/ppdb-2.0-xs-lexical
 node $srcdir/tool/genie.js augment paraphrase1.tsv $srcdir/test/data/en-US/synthetic.tsv --thingpedia $srcdir/test/data/en-US/thingpedia.tt \
-  --ppdb compiled-ppdb.bin --parameter-datasets $srcdir/test/data/en-US/parameter-datasets.tsv \
+  --ppdb compiled-ppdb.bin -pl en --parameter-datasets $srcdir/test/data/en-US/parameter-datasets.tsv \
```

Comment:
Two letter short options are a really bad idea (because short options are meant to be merged, so `-pl` should be equivalent to `-p -l`). I didn't catch this earlier but please use long options (--) instead. You can abbreviate long options as long as they are unambiguous.

Reply:
fixed